boards/arm/blackpill_f411ce/blackpill_f411ce.dts

```diff
@@ -10,7 +10,7 @@
 
 / {
 	model = "WeAct Studio Black Pill V2.0 Board";
-	compatible = "weact,blackpill-f411ce","st,stm32f411";
```

Comment:
changes to `f411ce` board should go into a separate commit.

Reply:
I thought so too, which was originally what I had, but @erwango suggested that I combine the 2 commits, so I did. If you want I can split it back up.

Reply:
Did I ?
Ho got it! I thought this second commit was a bunch of fixes on the initial commit (as this often happened recently).
My bad, I was too fast. Yes 2 commits are required as these are different boards. Sorry @coderkalyan for the back and forth changes

Reply:
No worries, it happens :smiley: I'll split up the commits again soon.